src/Mono.Android/Mono.Android.csproj

```diff
@@ -361,6 +361,11 @@
       <Name>jcw-gen</Name>
       <ReferenceOutputAssembly>False</ReferenceOutputAssembly>
     </ProjectReference>
+    <ProjectReference Include="..\..\external\Java.Interop\tools\jnimarshalmethod-gen\Xamarin.Android.Tools.JniMarshalMethodGenerator.csproj">
```

Comment:
Why is this needed at all?

Reply:
Probably not needed. It was there to be sure that it is available at the right time. When I removed the part that pre-generated the marshal methods in Mono.Android I didn't want to break it, so rather kept it there.

OK, I guess I will try another round.